cypress/integration/inventory/assign-preceding-title.spec.js

```diff
@@ -0,0 +1,52 @@
+import TopMenu from '../../support/fragments/topMenu';
+import getRandomPostfix from '../../support/utils/stringTools';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import InventoryInstanceEdit from '../../support/fragments/inventory/InventoryInstanceEdit';
+
+describe('ui-inventory: Assign a Preceding title for an instance', () => {
+  const instanceTitle = `autoTestInstanceTitle.${getRandomPostfix()}`;
+  const instanceTitle2 = `autoTestInstanceTitle.${getRandomPostfix()}`;
+  const precedingTitleValue = 'Preceding title test value';
```

Comment:
Better to use getRandomPostfix() here
It helps to debug esilier if test will be failed in future